content/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent.md

```diff
@@ -250,23 +250,23 @@ If you are using macOS or Linux, you may need to update your SSH client or insta
    {% mac %}
 
    ```shell
-   > Enter a file in which to save the key (/Users/YOU/.ssh/id_{% ifversion ghae %}ecdsa{% else %}ed25519{% endif %}_sk): [Press enter]
+   > Enter a file in which to save the key (C:\Users\%Username%\.ssh\id_{% ifversion ghae %}ecdsa{% else %}ed25519{% endif %}_sk): [Press enter]
```

Comment:
I think that would be fair for the mock output prompts, though I think for input prompts such as this [line](https://github.com/github/docs/pull/30971/files/cd9b410288fadecae5e9cc8970cd177a1ba82fa8#diff-78f7a8d1b7682e487c4ba53816c7dda1935628f62d8d8326030ccf62525e83e4R181), it would be more intuitive to have system variables instead?

Reply:
You're quite right that my statement wouldn't apply for that particular example; however, the concern we have is that using the system variables for examples will be confusing for a user who doesn't recognise them, and we're losing the clarity we get from using placeholders instead. It's also theoretically possible that someone may have changed the variables from the default ones.

I'd still be very happy to accept this PR if you'd consider changing the instances of `YOU` to `YOUR_USERNAME`?